### PR TITLE
Valor controllers fix

### PR DESCRIPTION
**Pull request by AntarcticaByToto**
-set limits -> add function to set forward and reverse independently
-change PIDF slot on neo
-implemented getCurrent() function

### CODE REVIEW

File: src/main/cpp/controllers/ValorFalconController.cpp [hunk 1]
```diff
@@ -52,6 +52,18 @@ void ValorFalconController::setLimits(int reverse, int forward)
     motor->ConfigReverseSoftLimitEnable(true);
 }
 
+void ValorFalconController::setForwardLimit(int forward)
+{
+    motor->ConfigForwardSoftLimitThreshold(forward);
+    motor->ConfigForwardSoftLimitEnable(true);
+}
+
+void ValorFalconController::setReverseLimit(int reverse)
```
Comment (janhelgeson) [inline src/main/cpp/controllers/ValorFalconController.cpp:61]: Any reason not to make this setReverseLimit(int reverse, bool enabled)?

**Review by janhelgeson — COMMENTED** [review]
@AntarcticaByToto @mray190  Wouldn't we want the ability to enable and DISABLE the limits? It looks like if we wanted to dynamically turn on/off the limits, we cannot. ATM you can only enable the limit

**Review by janhelgeson — DISMISSED** [review]
LGTM

**Review by janhelgeson — COMMENTED** [review]
LGTM

File: src/main/cpp/controllers/ValorFalconController.cpp (hunk 1 above)
Comment (janhelgeson) [reply src/main/cpp/controllers/ValorFalconController.cpp:61]: Turning off limits are not needed. This is a 1 time-config item

File: .wpilib/wpilib_preferences.json [hunk 2]
```diff
@@ -2,5 +2,5 @@
     "enableCppIntellisense": true,
     "currentLanguage": "cpp",
     "projectYear": "2022",
-    "teamNumber": 6800
+    "teamNumber": 6803
```
Comment (mray190) [inline .wpilib/wpilib_preferences.json:5]: Change back to 6800

File: src/main/cpp/Robot.cpp [hunk 3]
```diff
@@ -10,14 +10,15 @@
 #include <frc/smartdashboard/SmartDashboard.h>
 #include <frc2/command/CommandScheduler.h>
 
-Robot::Robot() : drivetrain(this)
+Robot::Robot() /*: drivetrain(this),*/
```
Comment (mray190) [inline src/main/cpp/Robot.cpp:13]: Need to re-enable the drivetrain

**Review by mray190 — CHANGES_REQUESTED** [review]

**Review by mray190 — APPROVED** [review]

**Pull request merged**